Pod/Assets/Emission.js.meta

```diff
@@ -0,0 +1 @@
+%���>�'�A��%:1���>
```

Comment:
I saw this when I was doing my work on deployment too, might be a new thing in RN